### PR TITLE
Upgrade to 0.9.43

### PR DESCRIPTION
**Pull request by zktony**
Upgrade to 0.9.43.

### CODE REVIEW

**Comment by zktony** [comment]
@serhii-temchenko could you please generate the benchmark code?

**Comment by serhii-temchenko** [comment]
> @serhii-temchenko could you please generate the benchmark code?

Done. Pushed in the latest commit. Please take a look.

**Review by serhii-temchenko — APPROVED** [review]

File: runtime/src/weights/block_weights.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub mod constants {
 	parameter_types! {
 		/// Importing a block with 0 Extrinsics.
 		pub const BlockExecutionWeight: Weight =
-			Weight::from_ref_time(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(5_000_000));
+			Weight::from_parts(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(5_000_000), 0);
```
Comment (Gauthamastro) [inline runtime/src/weights/block_weights.rs:27]: Is it valid to have a zero-proof size here? I think changing headers and other block generation details might have some size. @zktony Did you compare it with other runtimes?

File: runtime/src/weights/extrinsic_weights.rs [hunk 2]
```diff
@@ -24,7 +24,7 @@ pub mod constants {
 	parameter_types! {
 		/// Executing a NO-OP `System::remarks` Extrinsic.
 		pub const ExtrinsicBaseWeight: Weight =
-			Weight::from_ref_time(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(125_000));
+			Weight::from_parts(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(125_000), 0);
 	}
```
Comment (Gauthamastro) [inline runtime/src/weights/extrinsic_weights.rs:28]: Same as above

File: runtime/src/xcm_config.rs [hunk 3]
```diff
@@ -112,7 +139,7 @@ pub type XcmOriginToTransactDispatchOrigin = (
 
 parameter_types! {
 	// One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
```
Comment (Gauthamastro) [inline runtime/src/xcm_config.rs:141]: Comment is not valid anymore

File: runtime/src/xcm_config.rs [hunk 4]
```diff
@@ -163,26 +167,36 @@ impl xcm_executor::Config for XcmConfig {
 	type AssetTransactor = XcmHelper;
 	type OriginConverter = XcmOriginToTransactDispatchOrigin;
 	type IsReserve = MultiNativeAsset<AbsoluteReserveProvider>;
-	type IsTeleporter = (); // Teleporting is disabled.
-	type LocationInverter = LocationInverter<Ancestry>;
+	type IsTeleporter = ();
+	type UniversalLocation = UniversalLocation;
+	// Teleporting is disabled.
```
Comment (Gauthamastro) [inline runtime/src/xcm_config.rs:172]: Misplaced comment

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: runtime/src/weights/block_weights.rs (hunk 1 above)
Comment (35359595) [reply runtime/src/weights/block_weights.rs:27]: this is what latest version of template is using: [here](https://github.com/substrate-developer-hub/substrate-parachain-template/blob/ebcc7250a829e5374f3aa19834b840c460741fd8/runtime/src/weights/block_weights.rs#L27)

File: runtime/src/weights/extrinsic_weights.rs (hunk 2 above)
Comment (35359595) [reply runtime/src/weights/extrinsic_weights.rs:28]: this is what latest version of template is using: [here](https://github.com/substrate-developer-hub/substrate-parachain-template/blob/ebcc7250a829e5374f3aa19834b840c460741fd8/runtime/src/weights/extrinsic_weights.rs#L27)

File: runtime/src/weights/block_weights.rs (hunk 1 above)
Comment (zktony) [reply runtime/src/weights/block_weights.rs:27]: Yes, other parachains are also utilizing the same value.

File: runtime/src/xcm_config.rs (hunk 3 above)
Comment (zktony) [reply runtime/src/xcm_config.rs:141]: done

File: runtime/src/xcm_config.rs (hunk 4 above)
Comment (zktony) [reply runtime/src/xcm_config.rs:172]: done

**Review by 35359595 — APPROVED** [review]

**Review by 35359595 — APPROVED** [review]

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**